index.js

```diff
@@ -14,14 +20,14 @@ module.exports = {
       debugMode = process.env.EMBER_ENV === 'development';
     }
 
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      if (debugMode) {
-        app.import('bower_components/plupload/js/moxie.js');
-        app.import('bower_components/plupload/js/plupload.dev.js');
-      } else {
-        app.import('bower_components/plupload/js/plupload.full.min.js');
-      }
+
+    if (debugMode) {
+      app.import('vendor/plupload/moxie.js');
```

Comment:
@kratiahuja Can I ask why this is changed to the `vendor/` folder instead of `bower_components` folder? Is this something to do with `app.import` specifically? I've noticed in a few addons that `vendor` and `node_modules/bower_components` are almost interchangeable? Does Ember CLI treat them all the same during build or something?